hack/update-codegen.sh

```diff
@@ -20,5 +20,48 @@ set -o pipefail
 
 source $(dirname $0)/../scripts/library.sh
 
+CONFIG_GENERATOR_DIR="${REPO_ROOT_DIR}/tools/config-generator"
+CONFIG_DIR="${REPO_ROOT_DIR}/config/prow"
+PROW_GCS="knative-prow"
+GENERATE_TESTGRID_CONFIG="true"
+GENERATE_MAINTENANCE_JOBS="true"
+PROJECT="knative-tests"
+PROW_HOST="https://prow.knative.dev"
+TESTGRID_GCS="knative-testgrid"
+KNATIVE_CONFIG="config_knative.yaml"
```

Comment:
I'd rather these be args to generate_config()?

Reply:
I'm not sure if it's gonna be any cleaner unless we make them named params

Reply:
I don't like that inputs are all global vars, because if someone misses one in a later change, it would be hard to detect.

Reply:
I agree with you, removed all global vars, but just invoke `go run` twice seems cleaner to me. We can revisit this if it ever grows

Reply:
Yes, I like this result the best.